Update submission_file.txt

VadimKachevski opened the pull request:
Test with gradle project.

VadimKachevski committed:
Update submission_file.txt

Avielyo10 commented:
See your test results here https://gradle.com/s/zifctxsxx3ffe

VadimKachevski commented:
so i doesn't work because we are re-arranging the Polynoms.
When I get : "3.1+2.4x^2-x^4"
I re-arrange it to look like : "-1.0x^4+2.4x^2+3.1" 
And the assert equals on strings returns false.
Do you want us to remove the arrangement?

VadimKachevski commented:
Also
Expected :comp(-1.0x^4+2.4x^2+3.1,0.1x^5-1.2999999999999998x+5.0)
Actual   :comp(3.1+2.4x^2-x^4,5+2x-3.3x+0.1x^5)

-3.3x+2x 
I add them toghter to make it -1.3x (or javas -1.299999999999x...)

Avielyo10 commented:
@VadimKachevski 
I'll update my tests so the rearrangement will work, don't change a thing :)

Avielyo10 commented:
@VadimKachevski 
https://gradle.com/s/itsts5lsmayg6

VadimKachevski committed:
Update submission_file.txt

Changing to main repo

VadimKachevski commented:
I changed it to my main repository, do i need to create new pull request? I can't merge the repo I forked.

Avielyo10 commented:
This is a **verifier** you should upload `submission_file.txt` to the moodle

VadimKachevski commented:
Sure, but to test the gradle I created a new repository and it was successful, now I deleted all the contents of the original repository and copied the test repository to the original and wanted to verify the repo, but when i updated the submission_file it didn't run the tests.
I tried the merge the forked repos to match yours but it wont let me, so do i need to delete the old fork, create new fork and create new pull request ?

Avielyo10 commented:
@VadimKachevski Exactly!
